runtime/src/qio/qio_formatted.c

```diff
@@ -3396,6 +3408,8 @@ int _ftoa(char* restrict dst, size_t size, double num, int base, bool needs_i, c
                    precision, style->uppercase, style->prefix_base,
                    &skip);
 
+  // printf(" ...\n---\n%s\n", dst);
```

Comment:
Did you mean to leave this in?  It looks like helper output.  There's another couple like this below, as well